src/ucp/core/ucp_worker.h

```diff
@@ -44,6 +44,17 @@ enum {
 };
 
 
+/**
+ * UCP iface flags
+ */
+enum {
+    UCP_WORKER_IFACE_FLAG_OFFLOAD_ACTIVATED = UCS_BIT(0), /**< UCP iface receives tag
+                                                               offload messages */
+    UCP_WORKER_IFACE_FLAG_ON_ARM_LIST       = UCS_BIT(1)  /**< UCP iface is on the arm
```

Comment:
do we need to clarify what is arm list ?

Reply:
yeah, maybe it's confusing with certain CPU architecture name..